src/actions/catalogActions.js

```diff
@@ -125,8 +125,6 @@ export function catalogLoadIndex(catalog) {
           catalogName: catalog.metadata.name,
           id: catalog.metadata.name,
         });
-
-        throw error;
```

Comment:
LGTM, getting rid of these throws makes sense. 

But made me dig a bit into how the error is actually handled in the front end, and there is no user feedback in case of an error.

I think that's because I changed up the way these catalogs get loaded (from loading all catalogs on the CatalogList screen to loading a single one on the AppList screen. ) but I didn't implement the error handling.

So now now on failure we see "no apps matched your query", but it'd be better to show some kind of error message here:

<img width="1392" alt="Screenshot 2020-07-24 at 12 15 49 PM" src="https://user-images.githubusercontent.com/455309/88360324-c796c200-cda7-11ea-9b22-b4c8e434f8e8.png">

Doesn't have to be tackled in this PR btw, but wanted to mention it

Reply:
Yep, saw that. Already got another PR on the way 👍